metagpt/rag/schema.py

```diff
@@ -34,16 +35,21 @@ class IndexRetrieverConfig(BaseRetrieverConfig):
 class FAISSRetrieverConfig(IndexRetrieverConfig):
     """Config for FAISS-based retrievers."""
 
-    dimensions: int = Field(default=0, description="Dimensionality of the vectors for FAISS index construction.")
+    dimensions: int = Field(
+        default=config.embedding.dimensions, description="Dimensionality of the vectors for FAISS index construction."
+    )
 
     _embedding_type_to_dimensions: ClassVar[dict[EmbeddingType, int]] = {
         EmbeddingType.GEMINI: 768,
-        EmbeddingType.OLLAMA: 4096,
```

Comment:
Is this an expected removal?

Reply:
Yes. as https://github.com/ollama/ollama/issues/651 mentioned, the output dimension of ollama embedding may not be 4096.

Reply:
Thank you for pointing that out, indeed need to consider backward compatibility.